src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,77 @@
 package core.basesyntax;
 
+import core.basesyntax.exceptions.FileNotCreateException;
+import core.basesyntax.exceptions.FileNotExistsException;
+import core.basesyntax.exceptions.FileNotOpenException;
+import core.basesyntax.exceptions.FileNotWriteException;
+import core.basesyntax.models.Operation;
+import core.basesyntax.models.OperationDataResult;
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String OPEN_FILE_ERROR_MESSAGE = "Can't open the file: ";
+    private static final String FILE_NOT_EXISTS_ERROR_MESSAGE = "Can't open the file: ";
+    private static final String FILE_NOT_CREATE_ERROR_MESSAGE = "Can't create the file: ";
```

Comment:
```suggestion
    private static final String FILE_NOT_CREATED_ERROR_MESSAGE = "Can't create the file: ";
```
Fix everything similar